src/modules/render.js

```diff
@@ -0,0 +1,26 @@
+import apiURL from './api.js';
+
+const scoreList = document.getElementById('scoreList');
+// const score = document.getElementById('score');
+
+export default async function renderHTML() {
+  await fetch(apiURL)
+    .then((response) => response.json())
+    .then((json) => {
+      scoreList.innerHTML = '';
```

Comment:
- [ ]  You should use `async` and `await` with the `fetch` method to send and receive data.

Reply:
@ishimwezachee Thank you for the review, for the fetch part should I call the async function and then await with the fetch? Please help me in this matter, I am lost at the moment.